.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,54 @@
+name: 🐛 Bug Report
+description: Create a report to help us improve Nixpacks.
+title: "bug: "
```

```suggestion
title: ""
```
Title should be left blank IMO.